config/default/configmap/kfservice.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
I wonder if it would make sense for each framework to have its own config to avoid coupling?

Reply:
+1, I have added the data model to the ConfigMap,  please take a look if makes sense. We should easily add other configs such as readiness/liveness probe, resources in future.